Source/Classes/Categories/NSString+MD5.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  NSString+MD5.swift
+//  PINRemoteImage
+//
+//  Created by Rodrigo Ruiz Murguia on 27/01/22.
+//  Copyright © 2022 Pinterest. All rights reserved.
+//
+
+import CryptoKit
+import Foundation
+
+extension NSString {
+
+    private func MD5(with format: String) -> NSString {
+        guard #available(iOS 13, *),
+              #available(tvOSApplicationExtension 13.0, *) else { return "" }
```

Comment:
Should we be throwing fatal errors instead of returning empty strings?

Reply:
done!
I also found a new way to generate data object from string that avoids optional, so one less check is needed